core-service/src/main/java/com/crmfoodestablishment/coreservice/service/MenuServiceImpl.java

```diff
@@ -0,0 +1,72 @@
+package com.crmfoodestablishment.coreservice.service;
+
+import com.crmfoodestablishment.coreservice.dto.MenuDto;
+import com.crmfoodestablishment.coreservice.entity.Menu;
+import com.crmfoodestablishment.coreservice.mapper.MenuMapper;
+import com.crmfoodestablishment.coreservice.repository.MenuRepository;
+import lombok.RequiredArgsConstructor;
+import org.mapstruct.factory.Mappers;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import java.util.List;
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class MenuServiceImpl implements MenuService {
+
+    private final MenuRepository menuRepository;
+    private final MenuMapper menuMapper = Mappers.getMapper(MenuMapper.class);
+
+    private MenuNotFoundException throwMenuNotFoundException(UUID uuid) {
```

Comment:
there is no sense to use this private method
imo it is better to move error message to constant + u can use String.format